src/quo2/components/tabs/account_selector.cljs

```diff
@@ -66,7 +69,7 @@
                   :margin-bottom 8}}
         label-text])
      [rn/view {:style (account-container-row background-color)}
-      [rn/view {:style account-emoji-container}
+      [rn/view {:style (account-emoji-container (theme/get-theme))}
```

Comment:
an oversight
account emoji component should be used here: https://www.figma.com/file/WQZcp6S0EnzxdTL4taoKDv/Design-System-for-Mobile?type=design&node-id=483-19407&mode=design&t=O9RfvM3T2UYkamL3-4

There is this task here to update this component so either way this needs to be updated at some point here to correctly use this component - https://github.com/status-im/status-mobile/issues/16761

Reply:
@smohamedjavid can you fix this in your pr?

Reply:
Sure thing 👍